packages/core/src/lib/core/crud/crud.controller.ts

```
@@ -57,7 +57,7 @@ export abstract class CrudController<T extends BaseEntity> {
 		description: 'Records retrieved successfully'
 	})
 	@Get('pagination')
-	async pagination(@Query() filter?: PaginationParams<T>, ...options: any[]): Promise<IPagination<T> | void> {
+	async pagination(@Query() filter?: BaseQueryDTO<T>, ...options: any[]): Promise<IPagination<T> | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)